docs/pipeline-configuration-files/getting-started.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: 'Getting Started with Pipeline Configuration Files'
+sidebar_label: "Getting Started"
+sidebar_position: 0
+---
+
+Pipeline configuration files give you the ability to define pipelines that are provisioned by Conduit at startup.
+It's as simple as creating a YAML file that defines pipelines, connectors, processors, and their corresponding
+configurations.
+
+## Example pipeline (file to file)
+
+Create a folder called `pipelines` at the same level as your Conduit binary. Inside of that folder create a file named
+`file-to-file.yml`.
+
+```shell
+│ # Conduit binary
+├── conduit
+│ # Folder with pipeline configurations
+└── pipelines
+    │ # Pipeline configuration file
+    └── file-to-file.yml
+```
+
+Copy following content into `file-to-file.yml`:
+
+``` yaml title="pipelines/file-to-file.yml"
+version: 2.0
+pipelines:
+  - id: file-to-file
+    # run pipeline on startup
+    status: running
+    description: >
+      Example pipeline reading from file "example.in" and writing into file "example.out".
+
+    connectors:
+      - id: example.in
+        type: source
+        # use the built-in file plugin as the source
+        plugin: builtin:file
+        settings:
+          # read data from example.in
+          path: ./example.in
+
+      - id: example.out
+        type: destination
+        # use the built-in file plugin as the destination
+        plugin: builtin:file
+        settings:
+          # write data to example.out
+          path: ./example.out
+          # output the raw payload as a string
+          sdk.record.format: template
+          sdk.record.format.options: '{{ printf "%s" .Payload.After }}'
+```
+
+Now start Conduit and notice the log line saying that the pipeline `file-to-file` was created:
```

Comment:
I'd tweak this slightly. Maybe say something like `Now start Conduit. You should see a log line saying...`